rules/windows/lateral_movement_remote_file_copy_hidden_share.toml

```diff
@@ -34,8 +34,11 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-  process.name : ("cmd.exe", "powershell.exe", "robocopy.exe", "xcopy.exe") and
-  process.args : ("copy*", "move*", "cp", "mv") and process.args : "*$*"
+  (
+    process.name : ("cmd.exe", "powershell.exe", "xcopy.exe") and
+    process.args : ("copy*", "move*", "cp", "mv") or
+    process.name : "robocopy.exe"
+  ) and process.args : "*\\\\*\\*$*"
```

Comment:
The `process.args : "*$*"` condition is causing a lot of unintended matches, as the `$` chars are heavily used for other purposes like variables in PowerShell and file names. I am replacing it with a pattern that should match only hidden shares.

Reply:
Performance wise, does it make sense to rely on EQL's built in regex?

`process.args regex~ "\\\\[^\s\\]+\\[^\s\\]+\$.*"

Reply:
I am not sure if there is any benchmark comparing them. but shouldn't cause a lot of impact as they need to match the first condition before the wildcard